### PR TITLE
How to Visualize metrics using Grafana

### PR DESCRIPTION
**Pull request by frozenprocess**
## Description
This tutorial will guide user how to visualize the metrics with Grafana.

## Todos
Information about charts and statistics to Grafana.
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 210a28ff92ca15346941e2752adee49015ed6a64

https://deploy-preview-3472--calico.netlify.app

**Comment by frozenprocess** [comment]
This pull request is not ready yet!
I have submitted this to ask people about ideas for the dashboard.

**Comment by frozenprocess** [comment]
This is a followup on [last pull request "create Prometheus metrics"](https://github.com/projectcalico/calico/pull/3434).
I believe it's ready for review. 

Thanks in advance!
@hogepodge @lxpollitt @lwr20

File: _data/navbars/maintenance.yml [hunk 1]
```diff
@@ -13,7 +13,9 @@ section:
   path: /maintenance/monitor/
   section: 
   - title: Monitor component metrics
-    path: /maintenance/monitor-component-metrics  
+    path: /maintenance/monitor/monitor-component-metrics  
+  - title: Visual component metrics
```
Comment (bmckercher123) [inline _data/navbars/maintenance.yml:17]: Visualize component metrics

File: maintenance/monitor/monitor-component-visual.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+description: Use open source Grafana for visualizing Calico components.
+---
+
+### Big picture
+
+Use Grafana dashboard to generate visual aids related to the health of your {{ site.prodname }}
+
+### Value
+
+Using Grafana can be beneficial for IT personel, graphs and metrics can be crucial for you in order to identify any unsual activity.
```
Comment (hogepodge) [inline maintenance/monitor/monitor-component-visual.md:12]: "Using Grafana can be beneficial for IT personnel by providing a means to visualize metrics through graphs that can help you quickly identify unusual activity."

File: maintenance/monitor/monitor-component-visual.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+This how-to guide uses the following {{site.prodname}} features:
+
+**Felix** and **Typha** components configured with Prometheus configuration parameters (for consumption by Prometheus) and Grafana (for graphs and visual dashboards).
+
+### Concepts
+
+#### About Grafana
+
+Grafana is an open source visualization and analytics software. It allows you to query, visualize, alert on, and explore your metrics no matter where they are stored.
```
Comment (hogepodge) [inline maintenance/monitor/monitor-component-visual.md:24]: strike "your"

**Comment by hogepodge** [comment]
Grafana dashboard is asking for a password on my preview, I'm not sure what the defaults are or what the configuration is set to.

The large dashboard object isn't the way to go. We need to have this be a file that's hosted on the docs site and can be curl'ed down.

**Comment by frozenprocess** [comment]
> Grafana dashboard is asking for a password on my preview, we should mention the default is admin/admin.
@hogepodge : there is an info box stating what are default username and password for Grafana. maybe it would be better to bump it up a bit so user can see it more easily? 

https://github.com/projectcalico/calico/blame/5f73937c97f80015165627afdf692ac347db8358/maintenance/monitor/monitor-component-visual.md#L198-L201

File: maintenance/monitor/monitor-component-visual.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
+---
+title: Visualizing metrics via Grafana
+description: Use open source Grafana for visualizing Calico components.
+---
+
+### Big picture
+
+Use Grafana dashboard to generate visual aids related to the health of your {{ site.prodname }}
```
Comment (lxpollitt) [inline maintenance/monitor/monitor-component-visual.md:8]: Reads as an incomplete sentence

File: maintenance/monitor/monitor-component-visual.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+### Concepts
+
+#### About Grafana
+
+Grafana is an open source visualization and analytics software. It allows you to query, visualize, alert on, and explore metrics no matter where they are stored.
+
+### Before you begin...
+
+In this tutorial we assume that you have completed all steps in [component monitoring page]({{ site.baseurl }}/maintenance/monitor/monitor-component-metrics) and possess a running Kubernetes cluster with {{site.prodname}}, calicoctl and kubectl installed.
```
Comment (lxpollitt) [inline maintenance/monitor/monitor-component-visual.md:28]: in the monitor component metrics guide

File: maintenance/monitor/monitor-component-visual.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+This how-to guide uses the following {{site.prodname}} features:
+
+**Felix** and **Typha** components configured with Prometheus configuration parameters (for consumption by Prometheus) and Grafana (for graphs and visual dashboards).
+
+### Concepts
+
+#### About Grafana
+
+Grafana is an open source visualization and analytics software. It allows you to query, visualize, alert on, and explore metrics no matter where they are stored.
+
```
Comment (lxpollitt) [inline maintenance/monitor/monitor-component-visual.md:25]: Add about prometheus

File: maintenance/monitor/monitor-component-visual.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+This how-to guide uses the following {{site.prodname}} features:
+
+**Felix** and **Typha** components configured with Prometheus configuration parameters (for consumption by Prometheus) and Grafana (for graphs and visual dashboards).
+
+### Concepts
+
+#### About Grafana
+
+Grafana is an open source visualization and analytics software. It allows you to query, visualize, alert on, and explore metrics no matter where they are stored.
```
Comment (lxpollitt) [inline maintenance/monitor/monitor-component-visual.md:24]: including prometheus

**Pull request merged**